src/server.rs

```diff
@@ -52,3 +65,50 @@ impl Server {
         env_logger::try_init().context("init env logger")
     }
 }
+
+#[cfg(unix)]
+mod unix {
```

Comment:
Do we want to put this inside a new file?

Reply:
done